src/main/kotlin/org/wycliffeassociates/otter/common/data/workbook/Take.kt

```diff
@@ -11,5 +11,7 @@ data class Take(
     val number: Int,
     val format: MimeType,
     val createdTimestamp: LocalDate,
-    val deletedTimestamp: BehaviorRelay<Long?>
+    val deletedTimestamp: BehaviorRelay<DateHolder>
 )
+
+data class DateHolder(val value: LocalDate?)
```

Comment:
@sarabiaj The [RX Maybe](http://reactivex.io/RxJava/javadoc/io/reactivex/Maybe.html) isn't applicable here. We might have talked about [Arrow's Option](https://arrow-kt.io/docs/arrow/core/option/) or some implementation of a Haskell-style [Maybe](https://gist.github.com/ramseyboy/91bcc85cb5f28e47c9b5464c1501a983).  Those are pretty awesome, but their APIs for converting to/from nulls are more verbose than just using a data class.  I guess their opinion is that you should only rarely be using or converting nullables. That's sensible, but not the path we're on.